app/assets/stylesheets/govuk_publishing_components/components/_success-alert.scss

```diff
@@ -1,36 +1,9 @@
-.gem-c-success-alert {
-  color: $govuk-text-colour;
-  padding: govuk-spacing(3);
-  border: $govuk-border-width-narrow solid $govuk-success-colour;
-  @include govuk-responsive-margin(8, "bottom");
+@import "govuk/components/notification-banner/notification-banner";
 
-  @include govuk-media-query($from: tablet) {
-    padding: govuk-spacing(4);
-    border-width: $govuk-border-width;
-  }
+.gem-c-success-alert__content > * {
+  max-width: none;
```

Comment:
This is interesting, so I was taking a look at the Design System code that is this overwriting (below) I'm wondering if this max-width is there for an a11y reason.  I'm wondering if it's to prevent Content being too wide and harder to read. 🤔

`.govuk-notification-banner__content >* {
    -webkit-box-sizing: border-box;
    box-sizing: border-box;
    max-width: 605px
}`

Reply:
When I did this, it was to account for changes in the gem docs themselves to the component examples, but that doesn't make much sense! Removed.